modular-docs-manual/content/topics/module_modular-documentation-repositories.adoc

```diff
@@ -0,0 +1,141 @@
+[id="modular-documentation-repositories"]
+= Modular Documentation Repositories
+
+IMPORTANT: As CCS strategic imperatives and Pantheon 2 development continue to evolve, the details and recommendations in this section and possibly other sections in this guide might be modified. The Flexible Customer Content (FCC) team recognizes that the fluctuation in recommendations can be frustrating for CCS teams trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for CCS documentation.
+
+Due to many factors that determine documentation repository design for individual Red Hat products, the Flexible Customer Content (FCC) initiative cannot provide a strict documentation repository template for all CCS teams to follow. Each team faces different implementation challenges, especially when Red Hat product documentation is single-sourced with corresponding community project documentation.
+
+However, for optimal FCC, Pantheon, and Pantheon 2 compliance, the FCC initiative recommends a general repository structure similar to the following example, where possible:
+
+.Recommended documentation repository structure (where possible)
+[source]
+----
+├── _artifacts
+├── _images
+├── assemblies
+    ├── _artifacts -> ../_artifacts  // Symlinks can be extracted.
```

Comment:
What does 'extracted' mean in this context?

Reply:
I explain in the bulletted description that follows. Can do so with variables, build tooling, scripting, whatever. I was trying to avoid a long callout here, and thus provided more detail in the description that follows.